### PR TITLE
Adding  4.17 jobs

### PR DESCRIPTION
**Pull request by paigerube14**
Need to edit the cron schedules of these jobs but copied 4.16 aws jobs to 4.17

Running for sprint 454 4.next release

### CODE REVIEW

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.17-nightly-x86-data-path-9nodes

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.17-nightly-x86-control-plane-120nodes

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.17-nightly-x86-control-plane-120nodes periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.17-nightly-x86-data-path-9nodes

**Comment by paigerube14** [comment]
@afcollins are you able to help with providing good times/days for the cron for these jobs to run. I think we should be able to take the same cron configs as 4.16 had but would need to move 4.16 to another day. 
for the 255 sprint the selected build was from yesterday: https://amd64.ocp.releases.ci.openshift.org/releasestream/4.17.0-0.nightly/release/4.17.0-0.nightly-2024-06-25-162526 
TIA

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.17-nightly-x86-control-plane-120nodes

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.17-nightly-x86-data-path-9nodes

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.17-nightly-x86.yaml [hunk 1]
```diff
@@ -37,6 +37,171 @@ tests:
     - chain: openshift-qe-control-plane
     workflow: openshift-qe-installer-aws
   timeout: 2h30m0s
+- as: control-plane-120nodes
+  cron: 0 0 4,11,18,25 * *
```
Comment (afcollins) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.17-nightly-x86.yaml:41]: use: `0 2 4,11,18,25 * *` to balance

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.17-nightly-x86.yaml [hunk 2]
```diff
@@ -37,6 +37,171 @@ tests:
...
+    cluster_profile: aws-perfscale-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "24"
+      ZONES_COUNT: "3"
+    test:
+    - chain: openshift-qe-control-plane
+    workflow: openshift-qe-installer-aws
+- as: control-plane-etcdencrypt-24nodes
+  cron: 0 0 2,9,16,23 * *
```
Comment (afcollins) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.17-nightly-x86.yaml:72]: use: `0 3 2,9,16,23 * *` to balance

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.17-nightly-x86.yaml [hunk 3]
```diff
@@ -37,6 +37,171 @@ tests:
...
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "24"
+      ZONES_COUNT: "3"
+    test:
+    - chain: openshift-qe-router-perf
+    workflow: openshift-qe-installer-aws
+  timeout: 5h0m0s
+- as: node-density-heavy-24nodes
+  cron: 0 0 5,12,19,26 * *
```
Comment (afcollins) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.17-nightly-x86.yaml:111]: use: `0 3 5,12,19,26 * *` to balance

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.17-nightly-x86.yaml [hunk 4]
```diff
@@ -37,6 +37,171 @@ tests:
...
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "6"
+      POD_READY_THRESHOLD: 4s
+      ZONES_COUNT: "3"
+    test:
+    - chain: openshift-qe-node-density-cni
+    workflow: openshift-qe-installer-aws
+  timeout: 2h0m0s
+- as: data-path-9nodes
+  cron: 0 0 1,8,15,22 * *
```
Comment (afcollins) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.17-nightly-x86.yaml:153]: use: `0 3 1,8,15,22 * *` to balance

File: ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml [hunk 5]
```diff
@@ -2291,6 +2291,415 @@ periodics:
...
+    org: openshift-qe
+    repo: ocp-qe-perfscale-ci
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-perfscale
+    ci-operator.openshift.io/variant: aws-4.17-nightly-x86
+    ci.openshift.io/generator: prowgen
+    job-release: "4.17"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.17-nightly-x86-control-plane-120nodes
```
Comment (afcollins) [inline ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml:2312]: Please add slack reporter config in periodics file

Comment (paigerube14) [reply ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml:2312]: did someone take out all the reporter configs? I dont see any anymore in this file

Comment (afcollins) [reply ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml:2620]: I find them removed in commit ebe0c165c19491b12f1e3836209fd4c0e43564a3

**Comment by afcollins** [comment]
/hold pls I missed one of the jobs

also, w.r.t "I think we should be able to take the same cron configs as 4.16 had but would need to move 4.16 to another day" ... I've broken down distinct workloads to different days, then 4.16 vs. 4.17 would run at different times on the same day for a given workload.  There is different spreading depending on what you are looking for there.

If you want the "4.next" pattern, well there isn't any good way to do this..

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.17-nightly-x86.yaml [hunk 6]
```diff
@@ -37,6 +37,171 @@ tests:
...
+      SIZE_VARIANT: large
+      USER_TAGS: |
+        TicketId 382
+      ZONES_COUNT: "3"
+    test:
+    - chain: openshift-qe-control-plane
+    workflow: openshift-qe-installer-aws
+  timeout: 7h0m0s
+- as: control-plane-24nodes
+  cron: 0 2 2,9,16,23 * *
```
Comment (afcollins) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.17-nightly-x86.yaml:60]: Use: `0 3 2,9,16,23 * *`

**Comment by paigerube14** [comment]
> w.r.t

That sounds good to me if they are spread out and hopefully wont interefere with eachother. I mainly mentioned the 4.next that we can maybe spread out the 4.16 jobs to different days now. The main need is for the 4.next testing (now 4.17) to have a tuesday or wednesday run date so that I can report the results at the end of the sprint. Can put this as a future work item as we might be reducing the number of runs now since its been released

**Comment by afcollins** [comment]
Thanks Paige!

/lgtm

**Pull request merged**